themes/src/main/resources/theme/base/admin/resources/js/authz/authz-controller.js

```diff
@@ -1821,6 +1821,10 @@ module.controller('ResourceServerPolicyGroupDetailCtrl', function($scope, $route
                         return
                     }
                 }
+                if (group.id == "realm") {
```

Comment:
Should we check rather `(!group.path)` instead of `(group.id == realm)`? What if someone creates a group with id `"realm"`?

Reply:
@mhajas I'm not sure if that will be possible. The `realm` value should only be set by the UI when preparing the list of groups. See https://github.com/keycloak/keycloak/blob/5c7ce775cff43becdbaef98e53e482babc52dbcd/themes/src/main/resources/theme/base/admin/resources/js/controllers/groups.js#L3.

Reply:
@pedroigor I agree it should never happen and also this way it should be ok. However, the NPE is caused by null path sent from UI, so to avoid the NPE for 100% we should probably check for null/empty/undefined value in `group.path`. But this is just a detail, if you are happy with it this way I won't insist on changing it.